driver/format/ODBCDriver2.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include "driver/platform/platform.h"
+#include "driver/result_set.h"
+
+class ODBCDriver2ResultSet
```

Comment:
Could you please add a comment explaining what is the solve purpose of this `ResultSet` subclass and how it is different from any other `ResultSet` ?

Reply:
`ResultSet` is an abstract class, and this is an implementation of it, for `ODBCDriver2` format. Any comment that comes to my mind that could be put here, will rather mislead, than add to this fact. :) Please suggest, what text would help you to better understand this piece.

Reply:
I mean, it looks like it reads input data in some sort of textual format, you can basically state that and add a link to a format documentation for the reference.

Reply:
Will this be helpful:
```c++
// Implementation of ResultSet for ODBCDriver2 wire format of ClickHouse.
```